src/modules/Dashboard/features/RespondentData/RespondentDataHeader/RespondentDataHeader.tsx

```diff
@@ -86,6 +92,13 @@ export const RespondentDataHeader = ({
 
   const handleAssignActivity = () => {
     setShowActivityAssign(true);
+    Mixpanel.track({
+      action: MixpanelEventType.StartAssignActivityOrFlow,
+      [MixpanelProps.AppletId]: appletId,
+      [MixpanelProps.ActivityId]: activityId,
+      [MixpanelProps.EntityType]: 'activity',
```

Comment:
This header component could be showing in the context of an activity or a flow. This would be one way to handle it:
```suggestion
      [activityFlowId ? MixpanelProps.ActivityFlowId : MixpanelProps.ActivityId]: activityFlowId || activityId,
      [MixpanelProps.EntityType]: activityFlowId ? 'flow' : 'activity',
```